### ISSUE TITLE
Ubuntu Simple example,  Web server directives need updating

### ISSUE THREAD

**Issue by abuehrle** [weaveworks/guides#40]
web1 not joining the weave network as expected, in the sample: 

`sudo docker logs web1`

```
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 10.1.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 10.1.0.4. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 10.1.0.6. Set the 'ServerName' directive globally to suppress this message

```

**Comment by abuehrle** [comment]
http://askubuntu.com/questions/329323/problem-with-restarting-apache2

**Comment by abuehrle** [comment]
The apache image in this sample needs to updated.

**Comment by errordeveloper** [comment]
It's not a fatal error, just a warning, will try fixing later on.

**Comment by errordeveloper** [comment]
This also will probably get fixed in 1.1 with https://github.com/weaveworks/weave/pull/1265.
